examples/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java'
```

Comment:
also you need to add `checkstyle` plugin 

do not forget to copy/paste checkstyle plugin configurations
```
checkstyle {
    toolVersion '8.45'
    configDir rootProject.file('config')
    ignoreFailures false
    showViolations true
}
```

To verify run `./gradlew checkstyleMain`

Reply:
Done